views/account/profile/index.ejs

```diff
@@ -14,33 +25,32 @@
     >
       <div class="row col-12">
         <main class="px-3">
-          <h1 class="display-4">Goon up Brah.</h1>
-          <h1>Hello World!</h1>
-          <p>This file was served with Express.</p>
-          <p>Here is your information:</p>
-          <ul>
-            <li>First Name: <%= user.first_name %></li>
-            <li>Last Name: <%= user.last_name %></li>
-            <li>Email: <%= user.email %></li>
-            <li>Password: <%= user.password %></li>
-            <li>
-              Settings:
-              <ul>
-                <% var settings = JSON.parse(user.settings);
-                Object.keys(settings).forEach( (key) => { %>
-                <li><%= key %>: <%= settings[key] %></li>
-                <% }); %>
-              </ul>
-            </li>
-          </ul>
-
-          <a
-            href="#"
-            class="btn btn-warning btn-lg"
-            role="button"
-            aria-disabled="true"
-            >Generate</a
-          >
+          <h1 class="display-4">Profile</h1>
+          <h1 id="casual-header">Casual</h1>
+          <table id="casual-table">
+            <thead>
+                <tr>
+                    <th>Label</th>
+                    <th>Value</th>
+                    <th>Action</th>
+                </tr>
+            </thead>
+            <tbody id="casual-table-body">
+            </tbody>
+          </table>
```

Comment:
I'm also generally concerned about the usability of this on a mobile version, since this is the platform we're planning on demo'ing this with. The interface looks nice on the desktop but I'm not sure how well it resizes with the edit and delete buttons.

Reply:
Yeah valid; I was trying to make the interface as good as I could, even with the responsiveness of the jquery datatables. I'll try to play around with it at a later time to make it look more presentable. Anyone can hmu with ideas on how.